stripe/src/main/java/com/stripe/android/Stripe.kt

```diff
@@ -288,7 +292,7 @@ class Stripe internal constructor(
                 data,
                 ApiRequest.Options(
                     apiKey = publishableKey,
-                    stripeAccount = stripeAccountId
+                    stripeAccount = PaymentController.Result.fromIntent(data)?.stripeAccountId
```

Comment:
`StripePaymentController` already has the `publishableKey` and can extract the `stripeAccountId`, so let's create the `Options` instance there